api/src/services/bulk-get.js

```diff
@@ -9,7 +9,8 @@ const filterResults = (authorizationContext, result) => {
       if (!doc.ok) {
         return false;
       }
-      return authorization.allowedDoc(resultDocs.id, authorizationContext, authorization.getViewResults(doc.ok));
+      return authorization.allowedDoc(resultDocs.id, authorizationContext, authorization.getViewResults(doc.ok)) ||
+             authorization.isDeleteStub(doc.ok);
```

Comment:
I wonder if `isDeleteStub` should be part of the implementation of `allowedDoc` - is there any case when you'd want to call one but not the other?

Reply:
I thought about that as well. I decided to keep it separate to have the desired consistency of the changes feed. If I'd put it in `allowedDoc`, then the longpoll feeds would send these stubs to all offline users, which isn't harmful in any way, just generates more traffic then needed. 
If you prefer, it's little effort to include it in `allowedDoc`.

Reply:
Let's leave it as a separate function. Good thinking.